omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -362,4 +370,13 @@ def err_unused_postgres_configurable(setting)
       Server. Please check the release notes for details.
     EOM
   end
+
+  def err_unsupported_postgres_version(ver)
+    <<~EOM
+      Chef Server currently supports PostgreSQL version: #{SUPPORTED_VERSION}.
+      The database you have provided is running version #{ver}.
+
+      Please check the release notes for details.
```

Comment:
Anytime we tell folks to read the release notes we should link them to those release notes

```suggestion
      Please check the release notes at https://docs.chef.io/release_notes_server for details.
```